source/samples/pubsub/PubSubFeature.cpp

```diff
@@ -83,7 +153,8 @@ void PubSubFeature::publishFileData()
         LOG_ERROR(TAG, "Failed to read publish file... Skipping publish");
         return;
     }
-    auto onPublishComplete = [payload, this](Mqtt::MqttConnection &, uint16_t packetId, int errorCode) mutable {
+    auto onPublishComplete = [payload, this](Mqtt::MqttConnection &, uint16_t packetId, int errorCode) mutable
+    {
         LOGM_DEBUG(TAG, "PublishCompAck: PacketId:(%u), ErrorCode:%i", getName().c_str(), errorCode);
```

Comment:
There seems to be a mismatch in logging format strings.
* `%u` - this is `unsigned int`, but the type of `getName().c_str()` is a null-terminated string, suggest change to `%s`
* alternatively if the intent was to log the `packetId`, then replace `getName().c_str()` with `packetId` in the first position
 

https://man7.org/linux/man-pages/man3/printf.3.html

Reply:
done